tests/torch/requirements.txt

```diff
@@ -8,8 +8,8 @@ pytest-dependency>=0.5.1
 virtualenv
 
 # Required for search_building_blocks tests
-transformers[torch]~=4.36.0
-accelerate==0.24.1
+transformers[torch]~=4.38.2
+accelerate==0.28.0
```

Comment:
**accelerate** package was updated due to issues in the **transformers** package (issue no. 29216).